CHANGELOG.md

```diff
@@ -2,6 +2,53 @@
 
 All notable changes to this project will be documented in this file.
 
+## [v5.5.0](https://github.com/voxpupuli/puppetboard/tree/v5.5.0) (2024-11-26)
+
+[Full Changelog](https://github.com/voxpupuli/puppetboard/compare/v5.4.0...v5.5.0)
+
+**Breaking changes:**
+
+- Drop support for python 3.8 [\#1184](https://github.com/voxpupuli/puppetboard/pull/1184) ([TheMeier](https://github.com/TheMeier))
```

Comment:
because of this breaking change, it needs to be a major release (6.0.0).

Reply:
Ok, see #1200